grpc/src/main/java/com/linecorp/armeria/common/grpc/GrpcExceptionHandlerFunction.java

```diff
@@ -39,6 +39,14 @@ static GrpcExceptionHandlerFunctionBuilder builder() {
         return new GrpcExceptionHandlerFunctionBuilder();
     }
 
+    /**
+     * Returns the default {@link GrpcExceptionHandlerFunction}.
+     */
+    @UnstableApi
+    static GrpcExceptionHandlerFunction ofDefault() {
```

Comment:
Recently, we just use `of` for the default instance.
```suggestion
    static GrpcExceptionHandlerFunction of() {
```

Reply:
Applied feedback, Thanks.